cmake/os-macos.cmake

```diff
@@ -41,7 +42,7 @@ foreach(helper IN LISTS helper_suffixes)
     ${target_name}
     PROPERTIES MACOSX_BUNDLE_INFO_PLIST "${CMAKE_CURRENT_BINARY_DIR}/Info-Helper${helper_plist}.plist"
                OUTPUT_NAME "${target_output_name}"
-               FOLDER plugins/obs-browser/helper_suffixes
+               FOLDER plugins/obs-browser/Helpers
```

Comment:
Is this an unrelated fix or was that also required (not explained in the PR description)?